app/lib/server/functions/saveUser.js

```diff
@@ -170,6 +172,9 @@ export const saveUser = function(userId, userData) {
 		if (userData.email) {
 			createUser.email = userData.email;
 		}
+		if (userData.u) {
+			createUser.u = userData.u;
```

Comment:
Should there be some sort of authorisation check before this update? Should we allow a service account to update its owner?

Reply:
This is not update. This is when a user creates new account. In this file they used if else condition to check when it's update and when it's new user. This userData.u is inserted when service account  is created.